watchfiles/main.py

```diff
@@ -81,6 +81,13 @@ def watch(
         * otherwise, force polling is enabled
     * otherwise, we enable force polling only if we detect we're running on WSL (Windows Subsystem for Linux)
 
+    It is also possible change the poll delay between iterations, it can change to maintain a good response time
+    and an appropiate CPU consumption using the `poll_delay_ms` argument, we change poll delay thus:
+
+    * if file polling is enabled and the `WATCHFILES_POLL_DELAY_MS` environment variable exists and it is numeric:
+       * the `poll_delay_ms` argument value will change to the one defined in the environment variable.
+    * otherwise, it will be the original value of the argument
```

Comment:
```suggestion
    * if file polling is enabled and the `WATCHFILES_POLL_DELAY_MS` env var exists and it is numeric, we use that
    * otherwise, we use the argument value
```